openpype/lib/transcoding.py

```diff
@@ -1037,3 +1037,63 @@ def convert_ffprobe_fps_to_float(value):
     if divisor == 0.0:
         return 0.0
     return dividend / divisor
+
+
+def convert_colorspace(
+    input_path,
+    output_path,
+    config_path,
+    source_colorspace,
+    target_colorspace,
+    view=None,
+    display=None,
+    logger=None
+):
+    """Convert source file from one color space to another.
+
+    Args:
+        input_path (str): Path that should be converted. It is expected that
+            contains single file or image sequence of same type
+            (sequence in format 'file.FRAMESTART-FRAMEEND#.exr', see oiio docs)
```

Comment:
I would also mention an example of input string to make it more clear `big.1-3#.tif`